src/main/java/io/fabric8/elasticsearch/plugin/kibana/DocumentBuilder.java

```diff
@@ -60,20 +56,20 @@ public DocumentBuilder intervalName(String name) {
 	
 	public DocumentBuilder applicationFields(boolean use_cdm) {
 		if ( use_cdm )
-			return addField(FIELDS_FIELD, StringEscapeUtils.escapeJava(APPLICATION_CDM_FIELDS));
```

Comment:
these are the fields for the Kibana index mapping. this can be omitted and it'll still work?